code/modules/spacepods/spacepod.dm

```diff
@@ -284,35 +283,27 @@
 /obj/spacepod/emp_act(severity)
 	occupant_sanity_check()
 	cargo_hold.emp_act(severity)
+	
+	if(battery && battery.charge > 0)
+		battery.use((battery.charge / 2) / severity)
+	deal_damage(80 / severity)
```

Comment:
I've made some minor changes to the charge reduction/damage dealt by ion weapons here. The cell charge reduction now scales so two ion shots don't drain the entire pod, and the pod can now survive one more ion shot (a full ion gun will still kill it). These changes are based off of mech's.

I doubt the ion damage on pods was every properly tested (as it's been broken for a while now), which is why these numbers had to be tweaked a bit after testing.